Gemfile

```diff
@@ -1,8 +1,9 @@
 source 'https://rubygems.org'
 
 gem 'jbundler', platform: :jruby
-gem 'rails', '4.1.7'
-gem 'postgres_ext'
+gem 'rails', '4.1.9'
+gem 'postgres_ext', github: "edpaget/postgres_ext", branch: "dev"
```

Comment:
looks like your PR was merged and a new version released - https://github.com/dockyard/postgres_ext/pull/151